examples/BarChart.elm

```diff
@@ -0,0 +1,27 @@
+module OrdinalBands where
+
+import Svg exposing (Svg)
+import Plot exposing (..)
+import Plot.Scale as Scale
+import Plot.Axis as Axis
+
+main : Svg
+main =
+  let
+    xScale = Scale.ordinalBands ["a", "b", "c", "d"] (0, 800) 0.2 0.2
+    yScale = Scale.linear (0, 400) (800, 0) 10
```

Comment:
I think something weird is happening here. You are mapping `(0, 400)` to `(800, 0)`. If this is what is necessary to make the bars look like I would expect (0 is the bottom, 800 is the top) then I think there is something wrong. The plotting library should not require the user to map computer coordinates into expected coordinates.. I should be able to say that I want to map `(0, 400)` -> `(0,800)` and have the bars start at the bottom and end at the top.